spec/controllers/portal/teachers_controller_spec.rb

```diff
@@ -83,10 +83,11 @@
         
         post :create, params
         
-        assert_equal User.count(:all), current_user_count, "TeachersController#create erroneously created a User when given invalid POST data"
-        assert_equal Portal::Teacher.count(:all), current_teacher_count, "TeachersController#create erroneously created a Portal::Teacher when given invalid POST data"
-        #assert_not_nil flash.now[:error]
-        assert_nil flash[:notice]
+        expect(User.count(:all)).to eq(current_user_count), "TeachersController#create erroneously created a User when given invalid POST data"
```

Comment:
Is this valid `to` syntax?

Reply:
Yes I actually converted this (and one other) by hand because my regex did not handle it correctly. Thanks for checking though, I agree it appears odd. :smile: 

https://relishapp.com/rspec/rspec-expectations/v/3-8/docs/customized-message